engine/src/flutter/impeller/renderer/backend/gles/render_pass_gles.cc

```diff
@@ -583,8 +634,9 @@ bool RenderPassGLES::OnEncodeCommands(const Context& context) const {
   // resolved when we bind the texture to the framebuffer. We don't need to
   // discard the attachment when we are done.
   if (color0.resolve_texture) {
-    FML_DCHECK(context.GetCapabilities()->SupportsImplicitResolvingMSAA());
-    pass_data->discard_color_attachment = false;
+    pass_data->discard_color_attachment =
+        pass_data->discard_color_attachment &&
+        !context.GetCapabilities()->SupportsImplicitResolvingMSAA();
```

Comment:
I think the comment above needs to be updated.

Reply:
Done